src/devices/bus/snes/profighter.h

```diff
@@ -0,0 +1,82 @@
+// license:BSD-3-Clause
+// copyright-holders:Fabio Priuli
+
+#ifndef MAME_BUS_SNES_PROFIGHTER_H
+#define MAME_BUS_SNES_PROFIGHTER_H
+
+#pragma once
+
+#include "snes_slot.h"
+
+#include "formats/pc_dsk.h"
+#include "imagedev/floppy.h"
+#include "machine/upd765.h"
+
+
+class sns_pro_fighter_q_device : public device_t,
+						public device_sns_cart_interface
+{
+public:
+	// construction/destruction
+	sns_pro_fighter_q_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+protected:
+	// device-level overrides
+	virtual void device_start() override;
```

Comment:
Please don’t copy/paste this comment everywhere.  What does it even mean?  What‘s the device “level”?  What other “levels” are there?